ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml

```diff
@@ -88,6 +88,7 @@ tests:
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
 - as: perfscale-gcp-network-policy-5nodes-periodic
+  cron: 29 5 */1 * 1-5
```

Comment:
do you need to rebase with master, this was part of another pr corrrect

Reply:
I did rebase. May be it wasn't merged. Will try again.

Reply:
Odd, it looks like it placed it back

Reply:
something seems off here can you try to pull again? 
Can you also try to include this change I mentioned here: https://github.com/openshift/release/pull/43883/files#r1343021128